napari/layers/shapes/_tests/test_shapes_utils.py

```diff
@@ -1,10 +1,269 @@
 import numpy as np
+import pytest
+from numpy import array
 
 from napari.layers.shapes._shapes_utils import (
+    generate_2D_edge_meshes,
     get_default_shape_type,
     number_of_shapes,
 )
 
+W_DATA = [[0, 3], [1, 0], [2, 3], [5, 0], [2.5, 5]]
+
+cases = [
```

Comment:
I'm not happy about that long test/expected do you see any other ways to do that ?

Reply:
I'm afraid not really - i think this is ok though